code/core/src/components/components/Listbox/Listbox.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { type ComponentProps, forwardRef } from 'react';
```

Comment:
Also here: Public API, are we ready to committing to this?
Does this replace other components, should those be marked deprecated?

Should we write migration notes?
cc @shilman